packages/lib-classifier/src/plugins/tasks/survey/components/components/Chooser/components/Choices/components/ChoiceButton.js

```diff
@@ -26,10 +26,10 @@ function ChoiceButton({
 
   const choiceButton = useRef(null)
   const handleClick = useCallback(() => {
-    onChoose(choiceId)
+    !disabled && onChoose(choiceId)
```

Comment:
Disabled buttons can’t fire click events, so I’m curious as to how this gets triggered.

Reply:
By adding the `as='li` I think it's taking away some expected button behavior, including `disabled` functionality. For the related tests I added this line (and the similar line in `handleKeyDown`).

Reply:
You shouldn’t use `as=‘li’` on buttons! A list of buttons should look like this in HTML. A screen reader will announce this as a list with two items. Each item is a button. 

```html
<ul>
  <li><button>First Button</button></li>
  <li><button>Second Button</button></li>
</ul>
```

Reply:
Since the list can be navigated with the arrow keys, it behaves like a menu, so it might be appropriate to mark it up as a menu:

```html
<ul role=“menu”>
  <li role=“menuitem”>
    <button>First Button</button>
  </li>
  <li role=“menuitem”>
    <button>Second Button</button>
  </li>
</ul>
```

Reply:
Great suggestions, refactored accordingly!